XLForm/XL/Cell/XLFormSelectorCell.m

```diff
@@ -274,6 +283,8 @@ -(void)formDescriptorCellDidSelectedWithFormController:(XLFormViewController *)c
                                                                   }]];
             }
             [formViewController presentViewController:alertController animated:YES completion:nil];
+            // Fix for "Snapshotting a view that has not been rendered results in an empty snapshot." bug
+            [[alertController view] layoutIfNeeded];
```

Comment:
While running the code with Xcode 7.3.1 with iOS 9 iPad simulator, presenting the action sheet causes a bunch of "Snapshotting a view that has not been rendered results in an empty snapshot." warnings being shown at the Xcode console.